content/en/blog/falcosidekick-reponse-engine-part-6-cloud-run.md

```diff
@@ -0,0 +1,189 @@
+$ gcloud config set run/region us-west1
+$ gcloud config set run/platform managed
+$ gcloud run deploy $FUNCTION_NAME --image=$(ko publish .) \
+    --set-env-vars GKE_CLUSTER_NAME=$GKE_CLUSTER_NAME \
+    --set-env-vars PROJECT_ID=$PROJECT_ID
+...
+Allow unauthenticated invocations to [pod-deleter] (y/N)?  N
+
+Deploying container to Cloud Run service [pod-deleter] in project [developerguy-311909] region [us-west1]
+✓ Deploying... Done.
+  ✓ Creating Revision...
+  ✓ Routing traffic...
+Done.
+Service [pod-deleter] revision [pod-deleter-00002-cej] has been deployed and is serving 100 percent of traffic.
+Service URL: https://pod-deleter-uoz6q2wria-uw.a.run.app
+```
+
+### Install Falco + Falcosidekick
+
+Now, it is time to set up `Falco`, `Falcosidekick` with `Cloud Run` output type enabled.
+
+```shell
+$ kubectl create namespace falco
+$ helm repo add falcosecurity https://falcosecurity.github.io/charts
+$ helm repo update
+$ helm install falco falcosecurity/falco --namespace falco \
+--set ebpf.enabled=true \
+--set falcosidekick.enabled=true \
+--set falcosidekick.webui.enabled=true \
+--set falcosidekick.config.gcp.cloudrun.endpoint=$(gcloud run services list --format json | jq -r ".[] | select(.metadata.name==\"$FUNCTION_NAME\") | .status.address.url") \
+--set falcosidekick.config.gcp.cloudrun.jwt=$(gcloud auth print-identity-token)
+```
+
+Check the logs to see if `Cloud Run` output enabled for Falcosidekick.
+
+```shell
+$ kubectl logs -f falco-falcosidekick-7cd7bc6859-2nd9t --namespace falco
+falco-falcosidekick-7cd7bc6859-2nd9t falcosidekick 2021/06/07 16:03:14 [INFO]  : Enabled Outputs : [GCPCloudRun WebUI]
+falco-falcosidekick-7cd7bc6859-2nd9t falcosidekick 2021/06/07 16:03:14 [INFO]  : Falco Sidekick is up and listening on :2801
+...
+```
+
+If you see the `GCPCloudRun` in the list of enabled outputs, you consider that everything is good 👍.
+
+### Test
+
+We start by creating a dumb pod:
+
+```shell
+$ kubectl run alpine --namespace default --image=alpine --restart='Never' -- sh -c "sleep 600"
+pod/alpine created
+```
+
+```shell
+$ kubectl get pods --namespace default
+AME     READY   STATUS    RESTARTS   AGE
+alpine   1/1     Running   0          11s
+```
+
+Let's run a *shell* command inside and see what happens:
+
+```shell
+$ kubectl exec -i --tty alpine --namespace default -- sh -c "uptime"
+19:27:21 up 50 min,  load average: 0.11, 0.12, 0.11
+```
+
+As expected we got the result of our command, but, if we get the status of the pod we retrieve:
+
+```shell
+$ kubectl get pods --namespace default
+NAME     READY   STATUS        RESTARTS   AGE
+alpine   1/1     Terminating   0          103s
+```
+
+So, if you take a look at the logs of the `Cloud Run` function from the Google Cloud Console, you should see the following output:
+
+![cloud_run_function_output](/docs/images/cloud_run_function_outout.png)
+
+We can check the logs of the `Falco`, and `Falcosidekick` to see what happened.
```

Comment:
We can check the logs of the `Falco` and `Falcosidekick` to see what happened.

Reply:
thank you so much, done.